apps/crn-server/src/handlers/user/sync-active-campaign-contact.ts

```diff
@@ -23,32 +26,39 @@ const userDataProvider = getUserDataProvider();
 const assetDataProvider = getAssetDataProvider();
 const researchTagDataProvider = getResearchTagsDataProvider();
 
-const getContactPayload = async (
+export const getContactPayload = (
+  fieldIdByTitle: FieldIdByTitle,
   user: UserResponse,
-): Promise<ContactPayload> => {
-  const customFields = await getCustomFields(
-    activeCampaignAccount,
-    activeCampaignToken,
-  );
-
-  const fieldIdByTitle = getCustomFieldIdByTitle(customFields);
+): ContactPayload => {
+  const teams = user.teams.length
+    ? user.teams.flatMap((team, index) => [
+        {
+          field: fieldIdByTitle[`CRN Team ${index + 1}`]!, // eslint-disable-line @typescript-eslint/no-non-null-assertion
```

Comment:
this is a very strange assertion, how are you so sure that the value in the array with this index actually exists?

Reply:
Because I added this field manually in Active Campaign. The idea is to create a type for it then?